src/NServiceBus.Core/Installation/IInstall.cs

```diff
@@ -5,13 +5,12 @@ namespace NServiceBus.Installation
     /// <summary>
     /// Interface invoked by the infrastructure when going to install an endpoint.
     /// </summary>
-    public interface INeedToInstallSomething
+    public interface IInstall
     {
         /// <summary>
         /// Performs the installation providing permission for the given user.
         /// </summary>
         /// <param name="identity">The user for whom permissions will be given.</param>
-        /// <param name="config"><see cref="Configure"/> instance.</param>
-        Task InstallAsync(string identity, Configure config);
+        Task InstallAsync(string identity);
```

Comment:
Can be injected. Should we add it here explicitly so that we slowly remove the need for injecting?